resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -764,7 +764,8 @@ class="tw-whitespace-nowrap">{{ trans('item.filter.clear') }}</span>
                         </div>
                     </div>
                     <div
-                        class="tw-mx-auto tw-min-h-screen tw-max-w-screen-2xl tw-px-4 md:tw-px-8 md:tw-py-10">
+                        class="tw-mx-auto tw-min-h-screen tw-max-w-screen-2xl tw-px-4 md:tw-px-8 md:tw-py-10"
+                        v-if="!(artworks.length === 0 && isFetchingArtworks)">
```

Comment:
I'm having a hard time parsing this, I'm not even sure if this suggestion right:
```suggestion
                        v-if="artworks.length || isFetchingArtworks">
```